ReleaseHistory.md

```diff
@@ -1,4 +1,7 @@
 # SARIF Package Release History (SDK, Driver, Converters, and Multitool)
+## **v4.5.1 UNRELEASED
+* BUG: Report `WRN997_InvalidTarget` if the path of the scan target contains invalid character or encoded character.
+
```

Comment:
Add "an" before "invalid character" and before "encoded character"

Reply:
I change 'character' to 'characters' instead, as the encoded character could mean several character combination such as %2F.